core/src/trezor/crypto/slip39.py

```diff
@@ -485,67 +485,53 @@ def generate_random_identifier() -> int:
 
 
 def generate_single_group_mnemonics_from_data(
-    master_secret: bytes,
+    encrypted_master_secret: bytes,
     identifier: int,
     threshold: int,
     count: int,
-    passphrase: bytes = b"",
     iteration_exponent: int = DEFAULT_ITERATION_EXPONENT,
 ) -> List[str]:
     return generate_mnemonics_from_data(
-        master_secret,
-        identifier,
-        1,
-        [(threshold, count)],
-        passphrase,
-        iteration_exponent,
+        encrypted_master_secret, identifier, 1, [(threshold, count)], iteration_exponent
     )[0]
 
 
 def generate_mnemonics_from_data(
-    master_secret: bytes,
+    encrypted_master_secret: bytes,
     identifier: int,
     group_threshold: int,
     groups: List[Tuple[int, int]],
-    passphrase: bytes = b"",
     iteration_exponent: int = DEFAULT_ITERATION_EXPONENT,
 ) -> List[List[str]]:
     """
     Splits a master secret into mnemonic shares using Shamir's secret sharing scheme.
```

Comment:
Change comment to say: "Splits an encrypted master secret ..."

Reply:
We need to refactor it later properly anyway. But I've removed it.